src/main/kotlin/uk/gov/justice/digital/assessments/services/PredictorService.kt

```diff
@@ -95,13 +95,14 @@ class PredictorService(
         answers,
         "total_sexual_offences_child_image"
       )?.toInt(),
-      totalNonSexualOffences = getNonRequiredAnswer(answers, "total_non_sexual_offences")?.toInt(),
+      totalNonContactSexualOffences = getNonRequiredAnswer(answers, "total_non_sexual_offences")?.toInt(),
       earliestReleaseDate = getRequiredAnswer(answers, "earliest_release_date"),
       hasCompletedInterview = hasCompletedInterview,
       dynamicScoringOffences = getDynamicScoringOffences(hasCompletedInterview, answers)
     )
 
-    return assessRisksAndNeedsApiRestClient.getRiskPredictors(predictorType, offenderAndOffencesDto)
+    val final = true
```

Comment:
should this be hardcoded?

Reply:
just for now, I need to create 2 new endpoints outside the complete episode where we are calling the calculation from now. so we can call them from the UI and complete the flow to have both draft and final.